internal/kafka/internal/services/quota/internal/utils/simple_billing_model_resolver.go

```diff
@@ -0,0 +1,141 @@
+package utils
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+/********************************************************************************************************************
+ * This resolver is used when the user didn't specify the `billing model` and the `marketplace` in the kafka request.
+ * It implements the legacy logic: if the user has quota for `standard`, it returns `standard`. If he has quota for
```

Comment:
> It implements the legacy logic

Why legacy? Is this code meant to be removed at some point in the future?

Reply:
See previous comment